packages/DatePicker/stories/DatePicker.stories.js

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import moment from "moment";
+import { storiesOf } from "@storybook/react";
+import L10n from "@paprika/l10n";
+import Example from "./DatePickerExample";
+import DatePicker from "../src/DatePicker";
+
+storiesOf("DatePicker", module)
+  .add("DatePicker", () => <Example locale="en" />)
+  .add("DatePicker with locale", () => <Example locale="ja" />)
+  .add("DatePicker with initialDate", () => <Example initialDate={moment("2019-01-01")} />)
+  .add("DatePicker with DatePicker.Input", () => (
```

Comment:
"DatePicker with format and DatePicker.Input" ?
(or maybe make two separate stories 🤷‍♂️ not a big deal for now though).

Reply:
I can create a story and put format into knobs!

Reply:
In the next PR, I guess?